requirements/serve.txt

```diff
@@ -1,3 +1,4 @@
 gradio
 protobuf
 tritonclient[grpc]
```

Comment:
we can also remove tritonclient[grpc] since triton is removed

Reply:
> we can also remove tritonclient[grpc] since triton is removed

 i have remove tritonclient[grpc] from serve.txt